Sources/ClientRuntime/Networking/Http/CRT/HTTP2Stream+ByteStream.swift

```diff
@@ -19,13 +20,24 @@ extension HTTP2Stream {
     /// There is no recommended size for the data to write. The data will be written in chunks of `manualWriteBufferSize` bytes.
     /// - Parameter body: The body to write
     /// - Throws: Throws an error if the write fails
-    func write(body: ByteStream) async throws {
+    func write(body: ByteStream, telemetry: HttpTelemetry, serverAddress: String) async throws {
+        let context = telemetry.contextManager.current()
+        var bytesSentAttributes = Attributes()
+        bytesSentAttributes.set(key: HttpMetricsAttributesKeys.serverAddress, value: serverAddress)
         switch body {
         case .data(let data):
             try await writeChunk(chunk: data ?? .init(), endOfStream: true)
+            telemetry.bytesSent.add(
+                value: data?.count ?? 0,
+                attributes: bytesSentAttributes,
+                context: context)
         case .stream(let stream):
             while let data = try await stream.readAsync(upToCount: manualWriteBufferSize) {
                 try await writeChunk(chunk: data, endOfStream: false)
+                telemetry.bytesSent.add(
+                    value: data.count,
+                    attributes: bytesSentAttributes,
+                    context: context)
```

Comment:
Telemetry needs to be here since this is where the chunks are written, for the `bytes_sent` metric